Cogs/StudentCommands.py

```diff
@@ -170,7 +168,7 @@ async def roll(self, ctx, *options):
         attempt are logged.
 
         Args:
-            *options(str): Size of dice being rolled. Should be input as 'd{number}'.
+            *options(str): Input to dice parser. String of various different forms
```

Comment:
This would be `tuple(str)` not `str`, wouldn't it?

Also I don't think it should have the asterisk

Reply:
Resolved by 66886ce